### ISSUE TITLE
Move unused / old / outdated pages into some kind of archive

### ISSUE THREAD

**Issue by jasminechatfield** [GenderedIntelligence/genderedintelligence.co.uk#12]
There are a lot of duplicates that have been made to change the URL e.g. /families/sessions and /families/group are similar

Older versions should be cleaned up, I would probably put them in an /archive folder but let me know if that's a bad idea for some reason

**Comment by kimadactyl** [comment]
I've made a ticket with issues for the GI team to address, I'll add this there.

**Comment by jasminechatfield** [comment]
If we wanted to use Siteleaf we'd have to do a sweep of it ourselves though
to get it below the 100 page limit

**Comment by kimadactyl** [comment]
yeah we shouldn't be making content decions based on free tier limits :)

also there are 477 files total lol
